contrib/win32/win32compat/w32log.c

```diff
@@ -85,14 +86,16 @@ syslog_etw(int priority, const char *format, const char *formatBuffer)
 		free(w_payload);
 }
 
-
 /*
  * log file location will be - "%programData%\\openssh\\logs\\<module_name>.log"
  */
 void
 openlog_file()
 {	
-	if (logfd != -1)
+	if (strcmp(identity, "sshd") == 0 && logfd != -1)
+		return;
+	
+	if (strcmp(identity, "sftp-server") == 0 && sftp_server_logfd != -1)
```

Comment:
rewrite this logic.. 

if (strcmp(identity, "sftp-server") == 0 && sftp_server_logfd != -1)
     return;
else if (logfd != -1)
     return;

Reply:
This logic will fail if the identity is "sftp-server" and sftp_server_logfd == -1. It will return on the second if statement.